packages/shared/src/lib/user.ts

```diff
@@ -231,6 +231,7 @@ export enum LogoutReason {
 }
 
 export enum ContentLanguage {
+  Disabled = 'disabled',
```

Comment:
When I set the value to be `null`, it added an extra value to the enum on the JS side :sadcat: